src/components/modal/modal.tsx

```diff
@@ -419,6 +420,7 @@ export class Modal implements ConditionalSlotComponent, OpenCloseComponent {
   @Watch("open")
   async toggleModal(value: boolean): Promise<void> {
     this.active = value;
+    onToggleUnanimatedComponent(this);
```

Comment:
This will have to be called on all open/close components.

On a related note, it'd be great if we had a single way for handling both the duration > 0 and duration === 0 cases. We can leave this as a future refactor to not hold this up, but in that case, let's make it clear in the util's doc that it is required and where it should be called.